fixtures/assertion-removal/sku.config.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore TS1479
```

Comment:
We'll definitely need to address this somehow (in another PR).

Reply:
Yeah, this was a bit of a struggle to get all the lint to pass. I don't think TSC is picking the correct tsconfig.json settings. It should exclude these files. It should probably also exclude the test files since these require the fixture SKU configs.